contracts/Content/MultipleRoyalties.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "../libraries/LibRoyalty.sol";
+
+abstract contract MultipleRoyalties{
+
+    /***************** Stored Variables *****************/
+    mapping(uint256 => LibRoyalty.Fee[]) private tokenRoyalties;
```

Comment:
To be more efficient, you can replace this mapping with two mappings: 

mapping(uint256 => address[]) royaltyReceivers;
mapping(uint256 => uint24[]) royaltyRates;

That way, in _getMultipleRoyalties(), you don't have to allocate an address[] and uint24[] and then iterate over the tokenRoyalties to copy it over individually. 

just do:
return (royaltyReceivers, royaltyRates)

Reply:
ofcourse, there are a bunch of changes in this contract after doing that.